server/src/OnDeviceComponent.ts

```diff
@@ -4,17 +4,18 @@ import * as express from 'express';
 
 import { RokuDevice } from './RokuDevice';
 import { ConfigOptions } from './types/ConfigOptions';
-import { OnDeviceComponentRequest, KeyPathBaseTypes, OnDeviceComponentBaseResponse } from './types/OnDeviceComponentRequest';
 import * as utils from './utils';
+import { ODCRequest, ODCCallFuncArgs, ODCRequestOptions, ODCBaseResponse, ODCGetValueAtKeyPathArgs, ODCGetValuesAtKeyPathsArgs, ODCObserveFieldArgs, ODCSetValueAtKeyPathArgs, ODCRequestTypes, ODCRequestArgs } from '.';
```

Comment:
just curious any reason why we switch to import from "." instead of specific path?

Reply:
No. Was the default import. Happy to change if we see some reason for one or the other

Reply:
i don't have any preference i could not find any reason not too i was thinking  putting dot might cause it to scan the file system to find a match